README.md

```diff
@@ -31,5 +31,5 @@ The [project homepage](http://janusgraph.org) contains more information on Janus
 
 ## Contributing
 
-Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
```

Comment:
@kottmann @mbrukman In this PR why is ./.github/CONTRIBUTING.md rooted at ./.github and not ./?